packages/terra-clinical-site/src/examples/item-display/Index.jsx

```diff
@@ -23,9 +23,9 @@ const ItemDisplayExamples = () => (
     <div id="version">Version: {version}</div>
     <Markdown id="readme" src={ReadMe} />
     <h2> Item Display </h2>
-    <PropsTable id="props" src={ItemDisplaySrc} />
+    <PropsTable id="props" src={ItemDisplaySrc} componentName="Item Display" />
     <h2> Comment Item Display </h2>
```

Comment:
lets remove `<h2>Comment Item Display</h2>` as the title is now included in the appropriate props-table